packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -7,6 +7,7 @@ const {
 } = require('@cumulus/db');
 const cmrjsCmrUtils = require('@cumulus/cmrjs/cmr-utils');
 const { constructCollectionId } = require('@cumulus/message/Collections');
+const { GranuleNotPublishedError } = require('@cumulus/errors');
```

Comment:
```suggestion
const { GranuleNotPublished } = require('@cumulus/errors');
```
https://github.com/nasa/cumulus/blob/master/packages/errors/src/index.ts#L135 I think it's supposed to be `GranuleNotPublished` 🤔